From 4bf2d34df1aed3b1803635bc198697c0922c112e Mon Sep 17 00:00:00 2001
From: Mark Salyzyn <salyzyn@google.com>
Date: Thu, 12 Sep 2019 14:06:44 -0700
Subject: [PATCH 545/830] ANDROID: CONFIG_MMC=m
X-NVConfidentiality: public

Signed-off-by: Mark Salyzyn <salyzyn@google.com>
Test: compile
Bug: 140652382
Change-Id: Ie6002cde4d964920c71edd347851df9ceec4ee0f
(cherry picked from commit 30f8bea960e42921156a401e8e8ddb80fa3ee5c6)
---
 arch/arm64/configs/gki_defconfig | 6 +++---
 arch/x86/configs/gki_defconfig   | 3 +--
 2 files changed, 4 insertions(+), 5 deletions(-)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
index 1cbc3a53e419..e85c3a67f768 100644
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -322,11 +322,11 @@ CONFIG_USB_CONFIGFS_F_FS=y
 CONFIG_USB_CONFIGFS_F_ACC=y
 CONFIG_USB_CONFIGFS_F_AUDIO_SRC=y
 CONFIG_USB_CONFIGFS_F_MIDI=y
-CONFIG_MMC=y
+CONFIG_MMC=m
 # CONFIG_PWRSEQ_EMMC is not set
 # CONFIG_PWRSEQ_SIMPLE is not set
-CONFIG_MMC_SDHCI=y
-CONFIG_MMC_SDHCI_PLTFM=y
+CONFIG_MMC_SDHCI=m
+CONFIG_MMC_SDHCI_PLTFM=m
 CONFIG_NEW_LEDS=y
 CONFIG_LEDS_CLASS=y
 CONFIG_LEDS_TRIGGERS=y
diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
index e3537416896c..059e1dda256b 100644
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -280,8 +280,7 @@ CONFIG_USB_CONFIGFS_F_FS=y
 CONFIG_USB_CONFIGFS_F_ACC=y
 CONFIG_USB_CONFIGFS_F_AUDIO_SRC=y
 CONFIG_USB_CONFIGFS_F_MIDI=y
-CONFIG_MMC=y
-# CONFIG_MMC_BLOCK is not set
+CONFIG_MMC=m
 CONFIG_NEW_LEDS=y
 CONFIG_LEDS_CLASS=y
 CONFIG_LEDS_TRIGGERS=y
-- 
2.17.1

